packages/core/content-manager/server/src/controllers/collection-types.ts

```diff
@@ -166,6 +166,7 @@ export default {
     // @ts-expect-error populate builder needs to be called with a UID
     const populate = await getService('populate-builder')(model)
       .populateFromQuery(permissionQuery)
+      .populateDeep(Infinity)
```

Comment:
I think this might not be necessary! as the media is being populated in the entity service